pkg/credentialprovider/aws/aws_credentials_test.go

```diff
@@ -58,12 +58,15 @@ func (p *testTokenGetter) GetAuthorizationToken(input *ecr.GetAuthorizationToken
 
 func TestEcrProvide(t *testing.T) {
 	registry := "123456789012.dkr.ecr.lala-land-1.amazonaws.com"
-	otherRegistries := []string{"private.registry.com",
+	otherRegistries := []string{
+		"private.registry.com",
 		"gcr.io",
 	}
 	image := "foo/bar"
 
 	provider := &ecrProvider{
+		region: "lala-land-1",
```

Comment:
I used it because it looks a bit like a real region, but isn't one. It's a old habit, to guard against those rare times when test code ends up running unexpectedly against production.